### PR TITLE
auro-lockup: migrate repo to latest generator build

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

**Resolves:** #7 

## Summary:

This PR updates the repo to use the latest WC-Generator build

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (upgrade)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +60,22 @@ Since the legacy bundle includes many polyfills that are not needed by modern br
 <script src="https://unpkg.com/@alaskaairux/auro-lockup@latest/dist/auro-lockup__bundled.es5.js" nomodule></script>
 ```
 
+## auro-lockup use cases
```
Comment (blackfalcon) [inline README.md:63]: Nope, none of this. Update to bring back content lost in the migration.

File: docs/index.html [hunk 2]
```diff
@@ -81,9 +81,9 @@ <h3>auro-lockup: default</h3>
   </main>
   <div id="assets">
     <link rel="stylesheet"
-      href="https://unpkg.com/@alaskaairux/design-tokens@3.0.0/dist/tokens/CSSCustomProperties.css" />
+      href="https://unpkg.com/@alaskaairux/design-tokens@undefined/dist/tokens/CSSCustomProperties.css" />
```
Comment (blackfalcon) [inline docs/index.html:84]: Oops, what happened here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -2,36 +2,44 @@
...
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
```
Comment (jason-capsule42) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Numbering got screwed up here

File: .github/settings.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ repository:
   # See https://github.com/apps/settings for app settings
 
   # A short description of the repository that will show up on GitHub
-  description: A standardized HTML custom element for the Alaska Airlines extended experience lockup
+  description: Custom element that ....
```
Comment (jason-capsule42) [inline .github/settings.yml:6]: Looks like the description got removed here.

File: README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# auro-lockup
+# Lockup
 
-`<auro-lockup>` is a [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) for Alaska Airlines extended experiences.
+`<auro-lockup>` is a [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) for the purpose of ...
```
Comment (jason-capsule42) [inline README.md:3]: Description got wiped away here

File: README.md [hunk 6]
```diff
@@ -33,16 +33,13 @@ Any update to the Auro Design Tokens will be immediately reflected with browsers
...
 ```
 
 **Reference component in HTML**
 
 ```html
-<auro-lockup>
-  <span slot="title">Product Name</span>
-  <span slot="subtitle">Powered by Partner Name</span>
-</auro-lockup>
+<auro-lockup>Hello World</auro-lockup>
```
Comment (jason-capsule42) [inline README.md:42]: Looks like this shouldn't have been changed?

File: docs/upstream.md [hunk 7]
```diff
@@ -42,15 +42,20 @@ upstream        https://github.com/AlaskaAirlines/auro-lockup.git (push)
 
 #### Sync upstream
 
-To sync your master brach with your upstream master branch, there are two ways you can do this.
-
-One way is to fetch and merge. This will work if your `main` branch shares the same history as the upstream brach sans merged updates since fork or last sync.
+To sync your `main` brach with the upstream `main` branch, there are two ways you can do this. The preferred method is to fetch and rebase. Please do not merge down.
```
Comment (jason-capsule42) [inline docs/upstream.md:45]:
```suggestion
To sync your `main` branch with the upstream `main` branch, there are two ways you can do this. The preferred method is to fetch and rebase. Please do not merge down.
```

Comment (jason-capsule42) [reply docs/upstream.md:45]: This probably is in the generator, @blackfalcon.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (blackfalcon) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: Nope, intended. You can do this and markdown will convert it to the correct numbers.

File: docs/upstream.md (hunk 7 above)
Comment (blackfalcon) [reply docs/upstream.md:45]: This is a typo in the upstream template in the generator. I addressed the typo and added it to the DoD branch.

**Comment by blackfalcon** [comment]
@jason-capsule42 are there any open issues on this? Can we merge this?

File: .github/settings.yml (hunk 4 above)
Comment (jason-capsule42) [reply .github/settings.yml:6]: Not sure why this got resolved without comment.

File: src/style.scss [hunk 8]
```diff
@@ -4,14 +4,14 @@
...
+@import '../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
 
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/core";
+@import '../node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
+@import '../node_modules/@alaskaairux/webcorestylesheets/dist/core';
 
 // Support for auroElement styles
-// @import "./node_modules/@alaskaairux/webcorestylesheets/dist/auroElement/auroElement";
+// @import '../node_modules/@alaskaairux/webcorestylesheets/dist/auroElement/auroElement';
```
Comment (jason-capsule42) [inline src/style.scss:14]: Does this still need to be here?

**Review by jason-capsule42 — COMMENTED** [review]
Comments added directly in files changed

**Comment by blackfalcon** [comment]
This PR was started with WC-Generator v3.1.1. So the plan is to merge this one big one and then upgrade again to 3.3.0 and republish.

**Pull request merged**